web/src/components/pages/contribution/speak/audio-web.ts

```diff
@@ -93,29 +93,14 @@ export default class AudioWeb {
     );
   }
 
-  private visualize() {
+  private analyze() {
     this.analyzerNode.getByteFrequencyData(this.frequencyBins);
 
-    let sum = 0;
-    for (var i = 0; i < this.frequencyBins.length; i++) {
-      sum += this.frequencyBins[i];
-    }
-
-    let average = sum / this.frequencyBins.length;
-
     if (this.volumeCallback) {
-      this.volumeCallback(average);
+      this.volumeCallback(Math.max(...this.frequencyBins));
```

Comment:
Getting the average volume of a frequency spectrum is pretty complicated, especially when the data represents logarithmic data on a linear scale. And the old code _definitely_ didn't do what it said it did 😅 I think finding the max dB value is actually a closer measure to the signal's power, so I changed it here.